src/org/usfirst/frc4904/robot/subsystems/CrateIO.java

```diff
@@ -0,0 +1,28 @@
+package org.usfirst.frc4904.robot.subsystems;
+
+import org.usfirst.frc4904.standard.subsystems.motor.Motor;
+import edu.wpi.first.wpilibj.command.Subsystem;
+import org.usfirst.frc4904.robot.RobotMap;
+import org.usfirst.frc4904.robot.commands.RollersRest;
+
+public class CrateIO extends Subsystem {
+	
+	public static final double INTAKE_SPEED = 1.0;
+	public static final double OUTTAKE_SPEED = -1.0;
+	public static final double AT_REST = 0.0;
+	
+	public CrateIO(Motor roller) {
+		RobotMap.Component.CubeIOroller = roller;
```

Comment:
When this constructor takes in a Motor, it will already be stored. Don't store it a second time.

Reply:
(remove that line)